test/parallel/test-timers-api-refs.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+// don't verify the globals for this test
+common.globalCheck = false;
+
+// try overriding global APIs to make sure
+// they're not relied on by the timers
+global.clearTimeout = assert.fail;
```

Comment:
I don't think I understand?

if any of:

```
setTimeout()
clearTimeout()
setInterval()
clearInterval()
setImmediate()
clearImmediate()
```

Are using the exported or global version for specific internal-only behavior and then someone overwrites them, there will be problems...?